actions/overlay_action.go

```diff
@@ -42,6 +43,13 @@ func (overlay *OverlayAction) Verify(context *debos.DebosContext) error {
 	if _, err := debos.RestrictedPath(context.Rootdir, overlay.Destination); err != nil {
 		return err
 	}
+
+	if len(overlay.Origin) == 0 {
+		if _, err := os.Stat(overlay.Source); err != nil {
```

Comment:
the default origin for overlays is the recipedir, you need to take that into account otherwise it will fail when the recipe is not in the cwd of debos. As the new tests nicely show ;)